src/index.ts

```diff
@@ -11,11 +11,11 @@ import {
   TemplateDefinition,
   SubstateRequirement,
   Substate,
-} from './providers/types'
-import {MetamaskTariProvider}  from "./providers/metamask";
-import { MetaMaskInpageProvider } from '@metamask/providers';
-import {WalletDaemonTariProvider, WalletDaemonParameters} from "./providers/wallet_daemon";
-import {TariPermissions} from "./providers/wallet_daemon";
+} from "./providers/types";
```

Comment:
try to avoid changing linter conventions in existing codebase
we shouldn't be touching this line